index.html

```diff
@@ -2544,17 +2584,29 @@ <h1>
     </h1>
 
     <p>
-A <a>DID resolver</a> is a software or hardware component with an API for
-resolving <a>DIDs</a> of at least one <a>DID method</a>. It executes the read
-operation for the <a>DID method</a> corresponding to the <a>DID</a> being
-resolved to obtain the authoritative <a>DID document</a>. For more information,
-see Section <a href="#read-verify"></a>.
+This section defines the inputs and outputs of the
+<a>DID resolution</a> and <a>DID URL dereferencing</a> functions. The
+exact implementation of these functions is out of scope for this specification,
+but some considerations for implementors are available in [[?DID-RESOLUTION]].
     </p>
 
-    <p>
-The interfaces and algorithms for resolving <a>DIDs</a> and dereferencing
-<a>DID URLs</a> are specified in [[?DID-RESOLUTION]].
-    </p>
+    <section>
+        <h2>
+DID Resolution
+        </h2>
+    </section>
+
+    <section>
+        <h2>
+DID URL Dereferencing
+        </h2>
+    </section>
+
+    <section>
+        <h2>
+Resolution Metadata
```

Comment:
This is not an accurate name for the section as it covers both input and output metadata for both resolution and dereferencing. Suggest keeping the proposed name while other pull requests are still in progress so as not to lose synchronization.

```suggestion
Metadata Structure
```